scripts/convert-qpp-to-measures.js

```diff
@@ -42,6 +36,30 @@ var CEHRT_ELIGABLE_IA_MEASURE_IDS = [
   'IA_PM_4',
   'IA_PSPA_16'
 ];
+// Utils
+var isInverseBenchmarkRecord = require('./../util/is-inverse-benchmark-record');
+// Data
+var benchmarksData   = fs.readFileSync(path.join(__dirname, './../data/historical-benchmarks/2015.csv'), 'utf8');
+var benchmarkRecords = parse(benchmarksData, {columns: BENCHMARK_CSV_COLUMNS, from: 4});
+// Commandline arguments
+var category = process.argv[2] || 'ia';
+// Variables
+var qualityIdToIsInverseMap = {};
+var qpp = '';
+
+benchmarkRecords.forEach(function(record) {
+  // NOTE: There is a special case, e.g. qualityId 378 EHR, which has decile
```

Comment:
@marimiyachi Figured it out. There was a conflict between EHR and Registry/QCDR versions of 378. As for qualityId 1, it was because '001' does not equal '1'. I had to normalize the id for correct lookup in the map.